actions/readFile_test.go

```diff
@@ -0,0 +1,121 @@
+package actions
+
+import "testing"
+
+func TestValidCommandGetRelationShipPositive(t *testing.T) {
+	inputString := "GET_RELATIONSHIP Satvy Sister-In-Law"
+	resultBool, _ := IsValidCommand(inputString)
```

Comment:
Let's not name the variables for their types. Maybe it can be just  `res` instead of `resultBool` and `cmd` instead of `inputString`.

Here is a good read:  https://dave.cheney.net/practical-go/presentations/qcon-china.html#_dont_name_your_variables_for_their_types

If you like, let's change this at other instances in the codebase too.